src/Umbraco.Infrastructure/PropertyEditors/ImageCropperPropertyEditor.cs

```diff
@@ -48,17 +48,17 @@ public class ImageCropperPropertyEditor : DataEditor, IMediaUrlGenerator
             ILocalizationService localizationService,
             IIOHelper ioHelper,
             IShortStringHelper shortStringHelper,
-            ILocalizedTextService localizedTextService)
+            ILocalizedTextService localizedTextService,
+            IImageUrlGenerator imageUrlGenerator)
             : base(logger, dataTypeService, localizationService, localizedTextService, shortStringHelper)
         {
             _mediaFileSystem = mediaFileSystem ?? throw new ArgumentNullException(nameof(mediaFileSystem));
             _contentSettings = contentSettings.Value ?? throw new ArgumentNullException(nameof(contentSettings));
             _dataTypeService = dataTypeService;
-            _localizationService = localizationService;
             _ioHelper = ioHelper;
 
             // TODO: inject?
-            _autoFillProperties = new UploadAutoFillProperties(_mediaFileSystem, logger, contentSettings);
+            _autoFillProperties = new UploadAutoFillProperties(_mediaFileSystem, logger, imageUrlGenerator);
```

Comment:
Not really related to this PR, but think it make sense to register UploadAutoFillProperties in the container and inject it instead of the arguments needed . Can be a separate PR :)